src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+    background-image: url(./../images/star.svg);
```

Comment:
Use the correct path for the star.svg image. Replace './../images/star.svg' with '../images/star.svg'.